packages/bpk-component-accordion/src/BpkAccordionItem.module.scss

```diff
@@ -57,7 +57,7 @@
   }
 
   &__title-text {
-    display: inline-block;
+    display: block;
```

Comment:
We _could_ leave this out as title-text is a child of a flex container it will default to `block`. Maybe it's good signposting? 

Although, what is the point of setting a `span` as a `block` level element? A span with `display: block` might as well be a `div`:

```
<div className={titleTextClassNames.join(' ')}>
```

If this change is made, we can remove this display value.